anti.md

```diff
@@ -7,45 +8,113 @@ good way to meet their needs.
 
 
 ### What's the need?
-_What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
-them?_
 
+Portal's registration system is a pain in the butt, and everyone knows it. From
+the student perspective, we want to
+
+    1. Be able to search for classes
+    2. Be able to register for classes
+    3. Be able to do the prior two things with low latency
+
+It's not a long list of requirements, but nevertheless the registration system
+makes searching for classes difficult, with an interact that doesn't allow
+students to easily search based on the criteria that they're interested in
+(professors, student availability, etc.). Additionally the system runs with
+high latency, and a lot of that is actually spent navigating the interface
+(selecting the right subject area, then waiting for the results, then showing
+all results, the waiting for all the results, etc.).
 
 ### Why a language?
-_Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+It's seems that a language might be a natural solution to the problem, as
+Portal currently suffers from an unwieldy interface and a lack of features. One
+could imagine designing a language allows users to do complex searches across
+classes and provides results to the in the way they want, the first time.
+
+One could also imagine extending that language to allow for class registration
+though it and by providing a scheduling assistant which resolves constraints.
 
 ### Why you?
-_What excites you about this idea? How did you come up with it?_
 
+As a Harvey Mudd student, I have a strong interest in a more effective
+registration system. I've also spent some time working on Casey Chu's scheduler
+application ([here](https://github.com/bitsofpancake/hmc-scheduler)), so I'm a
+bit familiar with the systems involved.
+
+I came up with this idea by asking students what made them feel the worst when
+they were interacting with computers, and someone brought up portal. Between
+that and the general complaints about Portal I feel confident this would be
+helping to solve a genuine problem.
```

Comment:
Agreed.  I don't think anyone has used the portal and thought, "Wow, this is really well-designed, intuitive, and pleasant to use!"